### PR TITLE
added .sass file support

### PR DESCRIPTION
**Pull request by iMoritz**
I added .sass (cleaner indent-sytax) file support using node-sass's *indentedSyntax* option

### CODE REVIEW

File: middleware.js [hunk 1]
```diff
@@ -12,13 +12,14 @@ var imports = {};
...
- *    `outputStyle` Sass output style (nested,expanded, compact or compressed)
- *    `response`    Always write output directly to response
+ *    `force`           Always re-compile
+ *    `debug`           Output debugging information
+ *    `src`             Source directory used to find .scss files
+ *    `dest`            Destination directory used to output .css files
+ *                      when undefined defaults to `src`.
+ *    `outputStyle`     Sass output style (nested,expanded, compact or compressed)
+ *    `indentedSyntax`  Use standard SCSS syntax (Sassy CSS) or the cleaner SASS syntax
+ *    `response`        Always write output directly to response
```
Comment (kevva) [inline middleware.js]: Could you reset the indentation on this?

File: middleware.js [hunk 2]
```diff
@@ -64,6 +65,10 @@ module.exports = function(options){
 
   var root = options.root || null;
 
+  var indentedSyntax = options.indentedSyntax || false;
+
```
Comment (kevva) [inline middleware.js]: Remove this empty line.

**Comment by kevva** [comment]
Could you add a test?

File: middleware.js [hunk 3]
```diff
@@ -64,6 +65,10 @@ module.exports = function(options){
 
   var root = options.root || null;
 
+  var indentedSyntax = options.indentedSyntax || false;
```
Comment (kevva) [inline middleware.js]: `var indentedSyntax = options.indentedSyntax || null;`

**Comment by iMoritz** [comment]
ok working on the test :)

**Comment by iMoritz** [comment]
implemented the test but it failes when running after the scss tests.. If i comment out the scss test it works just fine. Maybe you can help out :)

**Comment by jerone** [comment]
[**@iMoritz**](https://github.com/iMoritz) commented on [28 mrt. 2015 19:41 CET](https://github.com/sass/node-sass-middleware/pull/37#issuecomment-87281519 "2015-03-28T18:41:03Z - Replied by Github Comment Enhancer"):
> implemented the test but it failes when running after the scss tests.. If i comment out the scss test it works just fine. Maybe you can help out :)

Does the test still fail when you include some css into the test file? I remember a situation that `node-sass-middleware` failed when the files didn't had any content.

**Comment by am11** [comment]
@jerone. This is exactly what I thought when looking at the test code. :)

**Comment by am11** [comment]
Please upgrade package.json with node-sass v2.1.1 which is released today. Everything is exactly same as v2.0.1, except for an installation fix.

**Comment by iMoritz** [comment]
@jerone yeah .. was just a commit bug .. is fixed now 
@am11 updated package.json to node-sass 2.1.1

**Comment by am11** [comment]
@iMoritz, thanks! :)

In `package.json`, can you please upgrade the packages:

* connect: ^3.3.4 to ^3.3.5
* mocha: ^2.1.0 to ^2.2.1
* should: ^5.0.0 to ^5.2.0

(taken from https://david-dm.org/sass/node-sass-middleware#info=devDependencies&view=table)

File: test/middleware.js [hunk 4]
```diff
@@ -124,3 +126,102 @@ describe('Using middleware', function () {
...
+      var filesrc = fs.readFileSync(sassfile),
+          result = sass.renderSync({ data: filesrc.toString(), indentedSyntax: true });
+      request(server)
+        .get('/test.css')
+        .expect(result.css)
+        .expect(200, done);
+    });
+
+    it('writes the file contents out to the expected file', function (done) {
+      var filesrc = fs.readFileSync(sassfile),
```
Comment (am11) [inline test/middleware.js]: This test is failing on TravisCI.
Is it passing locally, when running `npm test`?

If so, please consider adding `this.timeout = 5000;` before line 168,

**Comment by am11** [comment]
Also, you can change `"mkdirp": "0.5.x",` to `"mkdirp": "^0.5.0",`

**Comment by iMoritz** [comment]
After some investigation I found out that node-sass cannot find the included file so this is the first problem.
Next I found out that the index.css file doesn't get recompiled even if turn on the *force* option in the middleware

**Comment by iMoritz** [comment]
After some investigation I found out that node-sass cannot find the included file so this is the first problem I fixed.
Next I found out that the index.css file only gets a recompile if *force* is turned on in the middleware

**Comment by iMoritz** [comment]
no idea why it failes on node v0.10.38 maybe someone could help out

**Comment by iMoritz** [comment]
will continue on this after the node-sass v3 update

**Pull request closed without merging**